dotnet/src/Functions/Functions.OpenApi.Extensions/Extensions/OpenApiKernelExtensions.cs

```diff
@@ -0,0 +1,160 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+
+using System;
+using System.Collections.Generic;
+using System.Diagnostics.CodeAnalysis;
+using System.IO;
+using System.Linq;
+using System.Net.Http;
+using System.Text.Json;
+using System.Threading;
+using System.Threading.Tasks;
+using Microsoft.Extensions.DependencyInjection;
+using Microsoft.Extensions.Logging;
+using Microsoft.Extensions.Logging.Abstractions;
+using Microsoft.OpenApi.ApiManifest;
+using Microsoft.OpenApi.Readers;
+using Microsoft.OpenApi.Services;
+using Microsoft.SemanticKernel.Http;
+
+namespace Microsoft.SemanticKernel.Plugins.OpenApi.Extensions;
+/// <summary>
+/// Provides extension methods for the <see cref="Kernel"/> class related to OpenAPI functionality.
+/// </summary>
+public static class OpenApiKernelExtensions
+{
+    /// <summary>
+    /// Imports a plugin from an API manifest asynchronously.
+    /// </summary>
+    /// <param name="kernel">The kernel instance.</param>
+    /// <param name="pluginName">The name of the plugin.</param>
+    /// <param name="filePath">The file path of the API manifest.</param>
+    /// <param name="executionParameters">Optional execution parameters for the plugin.</param>
+    /// <param name="cancellationToken">Optional cancellation token.</param>
+    /// <returns>The imported plugin.</returns>
+    [Experimental("SKEXP0099")]
```

Comment:
@SergeyMenshykh, is there anything I need to be aware of picking the ID for this? Can I pick the next increment of SKEXP?